backend/src/features/pizzas-providers/implementations/ormeau.class.ts

```diff
@@ -53,18 +46,14 @@ export class OrmeauProvider extends PizzasProvider {
   }
 
   getPrices(pizzaWrapper: Cheerio, $: CheerioStatic): number[] {
-    const pizzaPricesDom = pizzaWrapper.find('.prix');
-
-    return pizzaPricesDom.toArray().map(pizzaPriceDom => {
-      const price = $(pizzaPriceDom)
-        .children()
-        .remove()
-        .end()
-        .text()
-        .replace(',', '.');
+    const pizzaPricesText = pizzaWrapper
+      .find('.pizzap')
+      .text()
+      .replace(/[A-Za-z]+ /g, '')
+      .replace(/,/g, '.')
+      .trim();
 
-      return +price;
-    });
+    return pizzaPricesText.split(' ').map(text => +text);
```

Comment:
This might be simplified by

```
return pizzaWrapper
  .find('.pizzap')
  .replace(/,/g, '.')
  .match(/\d+(\.\d+)?/g)
  .map(Number);
```

Reply:
I simplified it, and it made the parser bug for the last (non existant) pizza... But that is good, because I fixed it so the last (non existant) pizza is not shown anymore \o/